dali/test/python/test_dali_video_plugin_decoder.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from nvidia.dali import pipeline_def
+import nvidia.dali.types as types
+import nvidia.dali.fn as fn
+
+import cv2
+import numpy as np
+
+import nvidia.dali.fn.plugin.video as video
```

Comment:
Just a general remark that in a few other places we tend to go for dali.plugin.sth,

![image](https://github.com/NVIDIA/DALI/assets/11878086/60d61c0c-5bee-427c-9d73-930df792eda3)

Reply:
I don't have a an opinion on it, tbh. @jantonguirao considered various options when we worked on the plugin build.

Regardless, I think this is out of scope for this PR. If needed, we can file a separate one that would change the location for the plugin. Is that ok?

Reply:
It was a conscious decision. We don't have a way to register operators outside `nvidia.dali.fn`. We would need to build those plugins completely separate, including the generation of fn module, etc, which I wanted to avoid. Therefore this choice